plugins/PostProcessingPlugin/scripts/InsertAtLayerChange.py

```diff
@@ -0,0 +1,55 @@
+# Created by Wayne Porter
+
+from ..Script import Script
+
+class InsertAtLayerChange(Script):
+    def __init__(self):
+        super().__init__()
+
+    def getSettingDataString(self):
+        return """{
+            "name": "Insert at layer change",
+            "key": "InsertAtLayerChange",
+            "metadata": {},
+            "version": 2,
+            "settings":
+            {
+                "insert_loc":
```

Comment:
I would not abbreviate the setting name:
```suggestion
                "insert_location":
```